docs/experience/longtasks.md

```diff
@@ -0,0 +1,63 @@
+  width=778
+/>
+
+
+## Page Load Details
+In addition to long tasks charts on the page loads overview, each individual page load details screen also contains information about long tasks detected during both hard and soft page loads. Each long task is described by its **name**, **timestamp**, and **duration**.
+
+The browser doesn't point us to a particular code in your application, but the **name** of the long task can point to an event of the page load that was executed for a longer period of time. The **name** value can take one of the following values:
+
+* **unknown** - the event was performed outside of the event loop, 
+* **self** - the event originated from an event loop task within the **browsing context**. A **browsing context** is an environment in which **Document** objects are presented to the user.
+* **same-origin-ancestor** - the long task originated from an event loop task within same-origin ancestor browsing context. A browsing context **A** is said to be an **ancestor** of another browsing context **B**, if the browsing context **B** is a child of the browsing context **A** or if there is a browsing context **C** that is a child of **A** and an ancestor of **B**. 
+* **same-origin-descendant** - the long task originated from an event loop task within same-origin descendant browsing context. A **descendant** browsing contexts of a **Document** is an ordered list of the nested browsing contexts.
+* **same-origin** - the long task originated from an event loop task within same-origin browsing context that is not ancestor or descendant,
+* **cross-origin-ancestor** - the same as **same-origin-ancestor** but happened within **cross-origin**, not same-origin,
+* **cross-origin-descendant** - the same as **same-origin-descendant** but happened within **cross-origin**, not same-origin,
+* **cross-origin-unreachable** - the same as **same-origin** but happened within **cross-origin**, not same-origin,
+* **multiple-contexts** - the event originated from an event loop task that involved multiple browsing contexts. 
+
+<img
+  class="content-modal-image"
+  alt="Page Load Details"
+  src="../../images/experience/longtasks/details.png"
+  title="Page Load Details"
+  width=778
+/>
+
+To learn more about various terms discussed here, please head to:
+
+* [browsing context](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context)
+* [same-origin](https://html.spec.whatwg.org/multipage/origin.html#same-origin)
+* [ancestor browsing context](https://html.spec.whatwg.org/multipage/browsers.html#ancestor-browsing-context)
+* [descendant browsing context](https://html.spec.whatwg.org/multipage/browsers.html#list-of-the-descendant-browsing-contexts)
+
+## Browser Support
+The following browsers support the Long Tasks API allowing Sematext Experience to gather the data about long tasks:
+
+* Google Chrome starting from version 58,
+* Microsoft Edge starting from version 79,
+* Opera,
+* Samsung Internet starting from version 7.0.
```

Comment:
Add \n?
Interesting, Safari and FF simply don't have this?  Wow.

Reply:
Adjusted.

Reply:
> Add \n?
> Interesting, Safari and FF simply don't have this? Wow.

Yeah, they don't. I was amazed until I started to work with the browser API, there are discrepancies all over the place. I still remember fighting with browser compatibility looooong time ago and I guess that never changed :)